Cargo.toml

```diff
@@ -10,3 +10,11 @@ edition = "2018"
 
 [dependencies]
 anyhow = ">=1.0"
+pkcs11 = ">=0.5.0"
+pkcs11-uri = ">=0.1.2"
```

Comment:
@stefanberger are these libraries using the same standard for PKCS11 URIs? This library says it's implementing RFC 7512, so it seems like there shouldn't be any issue here, but deferring to your opinion on the intricacies

Reply:
Following this page [here](https://nickray.github.io/pkcs11-uri/pkcs11_uri/index.html) it does:

```
PKCS#11 URI

Bare bones implementation of the RFC 7512 URI scheme for locating keys and other PKCS#11 objects.

This library is patched together from existing libraries, namely pkcs11, uriparse and percent-encoding, and is a work in progress.
```

So this library seems to want to do the same as mine does. Differences may be in the details.